processing-service-core/tenant-config-options.md

```diff
@@ -0,0 +1,194 @@
+# Overview of tenant specific config options
+
+## Setting tenant configurations
+
+A tenant can specify custom values for configuration that can then be used when creating their workflow for the enrichment process.
+A tenant can only specify a custom configuration value for a configuration that already has a global configuration option, if a global configuration option does not exist for the configuration that the tenant tries to set the service will return an APIException to make the user aware that a global configuration must be created first.
+If a tenant does not have a custom configuration set for the key provided in this call then one will be created for the tenant, if one does exist then its value will be updated with the value provided.
+
+There are two ways that a tenants custom configuration can be created, either per configuration, or as a bulk create request.
+
+### Per config creation:
+
+To create a config for a user the api requires several pieces of information to create the configuration and store them for the tenant, these are:
+
+1. **tenantId** - This is a unique string that identifies the tenant.
+2. **key** - This is the key to assign the custom configuration value to, this key must already have a corresponding global config registered in order to be accepted.
+3. **value** - This is the value to set for the custom configuration.
+
+A sample put request url would be as follows:
+
+    http://<host>:<port>/v1/tenants/{tenantId}/config/{key}
```

Comment:
Should it be `http://<host>:<port>/data-processing-service/v1/tenants/{tenantId}/config/{key}`?

Reply:
Yes, it should have been, it has been updated now to reflect this.